src/components/core/DpEditor/MentionList.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <ul class="o-list suggestion__popup">
+    <template v-if="items.length">
+      <li
+        class="o-list__item suggestion__list-item u-ph-0_5 u-pv-0_25"
+        :class="{ 'suggestion__list-item--is-active': index === selectedIndex }"
+        v-for="(item, index) in items"
+        :key="item.id"
+        @click="selectItem(index)">
+        <button class="btn--blank">
+          {{ item.label }}
+        </button>
+      </li>
+    </template>
+    <li class="o-list__item suggestion__list-item" v-else>
+      No result
+    </li>
+  </ul>
+</template>
+
+<script>
+export default {
+  name: 'MentionList',
+
+  props: {
+    items: {
+      type: Array,
+      required: true,
+    },
+
+    command: {
+      type: Function,
+      required: true,
+    },
+  },
+
+  data() {
+    return {
+      selectedIndex: 0,
+    }
+  },
+
+  watch: {
+    items() {
+      this.selectedIndex = 0
+    },
+  },
+
+  methods: {
+    onKeyDown({ event }) {
```

Comment:
Should we use a switch statement instead?

Reply:
shouldn't be part of the PR. I reverted everything with mentions reverted